pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -759,8 +759,14 @@ type Address struct {
 
 //END Video -------------------
 
+type Stats struct {
+	Period int `xml:"period,attr"`
+}
+
 type Ballooning struct {
```

Comment:
Perhaps this is an opportunity to change the device name to a MemBalloon, but I don't mind. :)  What do you think @rmohr ?

Reply:
Since these structures are IIUC intended to map pretty much 1:1 to libvirt XML elements, and libvirt uses `<memballon>` for this device, I think renaming it would make a lot of sense.

Reply:
I will include this in my PR.